notebooks/write_IdealLsIsisLabelIsisSpiceDriver.ipynb

```diff
@@ -0,0 +1,131 @@
+{
```

Comment:
It might be worth mentioning that `noproj` is the ISIS application typically used to make undistorted cubes in the notebook. (In case someone is trying to figure out how they might create an undistorted image that will be of the correct format to be read by the `IdealLsIsisLabelIsisSpiceDriver`)